classes/hotel.js

```diff
@@ -1,7 +1,23 @@
 class Hotel {
-    constructor() {
+  constructor(name, rooms, bookings) {
+    this.name = name;
+    this.rooms = rooms;
+    this.bookings = bookings;    
+  }
 
-    }
+  findReservationsByDate(date) {
+    return this.bookings.filter(booking => booking.date === date);
+  }
+
+  findAvailableRooms(date) {
+    const reservations = this.findReservationsByDate(date).map(res => res.roomNumber);
+
+    return this.rooms.filter(room => !reservations.includes(room.number))
+  }
```

Comment:
Your methods a short and easy to follow which is definitely showing good coder empathy.